dev-util/webstorm/Manifest

```diff
@@ -1 +1,2 @@
 DIST WebStorm-2017.2.4.tar.gz 257748171 BLAKE2B 2b0951e94f92a0d690bf9261573963dd6250d9b896784b7156dde8b39ec3a59f94da023bcd3183b404e4bfa16d27928d3eecedf54f5b9036e856af5237e8c5ed SHA512 283339f7757093affde698cfb751f7ced18d54159db7696c9fe6b4d37751e94ad1ca9f0c54db9d5ca256da35ee24454a3b48ced21d6db434c23ec94b023a4d09
+DIST webstorm-2019.1.191.6183.63.tar.gz 267639462 BLAKE2B 3a5a6a88104d65e31c21b3298d493b270f425ed7835eac2450c4f5bb7a8cee994d2b61c1448f86d3c8f436467cde3c525d88bf74cc9117e4a7c2ba0139d1a935 SHA512 8b6bd491041b061045e8ba8dbc950c534035617d510a7f8e59be17e7fa7d425843fb3dfde7a67f081a5125faa75302fbb21692648f2f686761f3b125f75b2f6a
```

Comment:
Could this whole ebuild be versioned like `webstorm-2019.1.191`, do the latter numbers change often, and are they packaged?